frontend/src/components/beta/common/CodeArgs.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { hexy } from "hexy";
+
+import { styled } from "../../../libraries/styles";
+
+import CodePreview from "../../utils/CodePreview";
+import JsonView from "./JsonView";
+
+const HexArgs = styled("div", {
+  padding: "10px 0",
+
+  "& > div": {
+    background: "#f8f8f8",
+    borderRadius: 4,
+    color: "#3f4246",
+    padding: 20,
+    fontSize: 14,
+    fontWeight: 500,
+
+    "& textarea, pre": {
+      background: "inherit",
+      color: "inherit",
+      fontFamily: "inherit",
+      fontSize: "inherit",
+      border: "none",
+      padding: 0,
+    },
+  },
+});
+
+const CodeArgs: React.FC<{ args: string }> = React.memo(({ args }) => {
+  const decodedArgs = Buffer.from(args, "base64");
```

Comment:
🤔 
I'm actually quite confused.

It seems like we're getting `Buffer` module from [`node-libs-browser`](https://github.com/webpack/node-libs-browser), but they are deprecated since webpack 5.
We're definitely using webpack 5 (checked version in next.config.ts in webpack options), but we have webpack 4 in our package-lock.

Most important thing is that works.